### ISSUE TITLE
Issue while running with device='auto'

### ISSUE THREAD

**Issue by NamburiSrinath** [zjunlp/EasyEdit#40]
Hi,

I'm trying to run GPT-J 6B with ROME and here's what I've done.

1. Changed the `model_name` in YAML file in `hparams/ROME/`
2. In `edit.py`, uncommented `test_ROME_GPTJ()`
3. In easyeditor/editors/editor/ added `self.model = AutoModelForCausalLM.from_pretrained(self.model_name, device_map='auto')`

When I run `python edit.py`, I got the following traceback!

```
Traceback (most recent call last):
RuntimeError: device >= 0 && device < num_gpus INTERNAL ASSERT FAILED at "../aten/src/ATen/cuda/CUDAContext.cpp":50, please report a bug to PyTorch. 

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/data/EasyEdit/edit.py", line 1860, in <module>
    main()
  File "/data/EasyEdit/edit.py", line 1817, in main
    test_ROME_GPTJ()
  File "/data/EasyEdit/edit.py", line 772, in test_ROME_GPTJ
    editor = BaseEditor.from_hparams(hparams)
  File "/data/EasyEdit/easyeditor/editors/editor.py", line 44, in from_hparams
    return cls(hparams)
  File "/data/EasyEdit/easyeditor/editors/editor.py", line 67, in __init__
    self.model = AutoModelForCausalLM.from_pretrained(self.model_name, device_map='auto')
...
torch.cuda.DeferredCudaCallError: CUDA call failed lazily at initialization with error: device >= 0 && device < num_gpus INTERNAL ASSERT FAILED at "../aten/src/ATen/cuda/CUDAContext.cpp":50, please report a bug to PyTorch. 
```

I've 8 GPUs and when I changed device: 0 or device: 4, I am still facing the same issue! 

Basically, I want to run using multiple GPUs as I believe ROME with GPT-J will take around 60-80GB. Please let me know how this can be done!

Thanks

**Comment by pengzju** [comment]
It is the holiday now,, and I will address the issues and get back to you after the holiday. I apologize for any inconvenience this may have caused.

**Comment by pengzju** [comment]
Thank you for your attention. The `accelerator` module is currently under active development.

**Comment by zxlzr** [comment]
The accelerator module has been added in EasyEdit. If you have any questions, feel free to contact us.
